<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DynamicRule</class>
 <widget class="QWidget" name="DynamicRule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Artist:</string>
       </property>
       <property name="buddy">
        <cstring>artistText</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="CompletionCombo" name="artistText"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Album Artist:</string>
       </property>
       <property name="buddy">
        <cstring>albumArtistText</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="CompletionCombo" name="albumArtistText"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Album:</string>
       </property>
       <property name="buddy">
        <cstring>albumText</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="CompletionCombo" name="albumText"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Title:</string>
       </property>
       <property name="buddy">
        <cstring>titleText</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="LineEdit" name="titleText"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Genre</string>
       </property>
       <property name="buddy">
        <cstring>genreCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="genreCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Year:</string>
       </property>
       <property name="buddy">
        <cstring>dateFromSpin</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSpinBox" name="dateFromSpin">
         <property name="minimumSize">
          <size>
           <width>72</width>
           <height>0</height>
          </size>
         </property>
         <property name="specialValueText">
          <string>Any</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="dateToSpin">
         <property name="minimumSize">
          <size>
           <width>72</width>
           <height>0</height>
          </size>
         </property>
         <property name="specialValueText">
          <string>Any</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;i&gt;&lt;b&gt;NOTE&lt;/b&gt; Only enter values for the tags you wish to be search on. &lt;/i&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>CompletionCombo</class>
   <extends>QComboBox</extends>
   <header>completioncombo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
